src/player/Player.php

```diff
@@ -1251,11 +1251,12 @@ protected function processMostRecentMovements() : void{
 
 			$horizontalDistanceTravelled = sqrt((($from->x - $to->x) ** 2) + (($from->z - $to->z) ** 2));
 			if($horizontalDistanceTravelled > 0){
-				//TODO: check swimming (adds 0.015 exhaustion in MCPE)
 				if($this->isSprinting()){
-					$this->hungerManager->exhaust(0.1 * $horizontalDistanceTravelled, PlayerExhaustEvent::CAUSE_SPRINTING);
+					$this->hungerManager->exhaust(0.01 * $horizontalDistanceTravelled, PlayerExhaustEvent::CAUSE_SPRINTING);
+				}elseif($this->isSwimming()){
```

Comment:
This is not sufficient. There's a reason that TODO was there.
`isSwimming()` only tells if the player toggled swimming (in the prone position), it doesn't account for wading on the surface of water.